tap_github/streams.py

```diff
@@ -195,6 +182,90 @@ def parse_response(self, response: requests.Response) -> Iterable[dict]:
     ).to_dict()
 
 
+class CommunityProfileStream(GitHubStream):
+    """Defines 'CommunityProfile' stream."""
+
+    name = "community_profile"
+    path = "/repos/{org}/{repo}/community/profile"
+    primary_keys = ["repo", "org"]
+    parent_stream_type = RepositoryStream
+    ignore_parent_replication_key = False
+    state_partitioning_keys = ["repo", "org"]
+    tolerated_http_errors = [404]
```

Comment:
this might be a good thing to add to the README too since a project could technically not have a readme, and it would halt here. Good catch!